src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -2323,8 +2323,7 @@ static void Main()
 
             var expr = GetExprSyntaxForBinding(GetExprSyntaxList(tree));
             var typeInfo = model.GetSymbolInfo(expr);
-            // lowest bound node with associated syntax is being picked up here ... fine for now.
```

Comment:
>// lowest bound node with associated syntax is being picked up here ... fine for now. [](start = 12, length = 85)

Perhaps the comment should be adjusted rather than removed. Something like: "the type info uses the type inferred for the first declared local".